core/templates/pages/exploration-player-page/services/state-classifier-mapping.service.spec.ts

```diff
@@ -36,7 +36,41 @@ describe('State classifier mapping service', () => {
       mappingService.init({
         stateName1: {
           algorithm_id: 'TestClassifier',
-          classifier_data: {},
+          classifier_data: {
```

Comment:
I think you can keep this object in a variable and use this variable in other places instead of asserting the hardcoded object.

Reply:
done